phpstan.neon.dist

```diff
@@ -1,4 +1,13 @@
-parameters:
-  ignoreErrors:
-    - '#@throws with type .+ is not subtype of Throwable#'
-  treatPhpDocTypesAsCertain: false
+parameters:
+  ignoreErrors:
+    -
+      message: '#@throws with type .+SimpleCache.+ is not subtype of Throwable#'
+      path: src/Repository/DefaultUnleashRepository.php
+      reportUnmatched: false
+    -
+      messages:
+        - '#Class .+Filesystem.* not found#'
+        - '#Class .+Flysystem.* not found#'
+      path: src/Helper/DefaultImplementationLocator.php
+      reportUnmatched: false
```

Comment:
These message will be reported if `cache/filesystem-adapter` is missing, and that's okay.

Reply:
Sure thing! Fixed via 4121a65.